hooks/test/browser/useState.test.js

```diff
@@ -234,6 +234,31 @@ describe('useState', () => {
 		expect(scratch.textContent).to.equal('hi');
 	});
 
+	it('correctly updates with multiple state updates', () => {
```

Comment:
I'd suggest to add a comment mentioning the original issue, since the sequence of updates needed to trigger the problem is quite specific, rather than this being just a general test for multiple states in one component.